articles/openshift/openshift-faq.md

```diff
@@ -10,196 +10,220 @@ ms.date: 11/04/2019
+For more information, see the OpenShift documentation on disabling self-provisioning for your cluster version:
 
-SDN is openshift-ovs-networkpolicy and is not configurable.
+- [Disabling self-provisioning in 4.3 clusters](https://docs.openshift.com/aro/4/applications/projects/configuring-project-creation.html#disabling-project-self-provisioning_configuring-project-creation).
+- [Disabling self-provisioning in 3.11 clusters](https://docs.openshift.com/container-platform/3.11/admin_guide/managing_projects.html#disabling-self-provisioning).
 
-## Which UNIX rights (in IaaS) are available for Masters/Infra/App Nodes?
+### Which UNIX rights (in IaaS) are available for Masters/Infra/App Nodes?
 
-Not applicable to this offering. Node access is forbidden.
+For 4.x clusters node access is available through the cluster-admin role. See [RBAC overview](https://docs.openshift.com/container-platform/4.3/authentication/using-rbac.html) for more details.
 
-## Which OCP rights do we have? Cluster-admin? Project-admin?
+For 3.11 clusters node access is forbidden.
 
-For details, see the Azure Red Hat OpenShift [cluster administration overview](https://docs.openshift.com/aro/admin_guide/index.html).
+### Which OCP rights do we have? Cluster-admin? Project-admin?
 
-## Which kind of federation with LDAP?
+For 4.x clusters, the cluster-admin role is available. See [RBAC overview](https://docs.openshift.com/container-platform/4.3/authentication/using-rbac.html) for more details.
 
-This would be achieved via Azure AD integration. 
+For 3.11 clusters, see the [cluster administration overview](https://docs.openshift.com/aro/admin_guide/index.html) for more details.
 
-## Is there any element in ARO shared with other customers? Or is everything independent?
+### Which identity providers are available?
 
-Each Azure Red Hat OpenShift cluster is dedicated to a given customer and lives within the customer's subscription. 
+For 4.x clusters you configure your own identity provider. See the Red Hat documentation on [configuring identity prodivers (https://docs.openshift.com/aro/4/authentication/identity_providers/configuring-ldap-identity-provider.html) for more details.
 
-## Can we choose any persistent storage solution, like OCS? 
+For 3.11 clusters you can use the Azure AD integration. 
 
-Two storage classes are available to select from: Azure Disk and Azure File.
+## Storage
 
-## How is a cluster updated (including majors and minors due to vulnerabilities)?
+### Is data on my cluster encrypted?
 
-See [What is the general upgrade process?](https://docs.microsoft.com/azure/openshift/openshift-faq#what-is-the-general-upgrade-process)
+By default, there is encryption at rest. The Azure Storage platform automatically encrypts your data before persisting it, and decrypts the data before retrieval. See [Azure Storage Service Encryption for data at rest](https://docs.microsoft.com/azure/storage/common/storage-service-encryption) for details.
```

Comment:
"By default, data is encrypted at rest."

Reply:
:+1: